smartdevicelink/sdl_java_suite
# Bugfix/issue_1343 foreground permissions exception

Fixes #1343 

This PR is **[ready]** for review.

### Risk
This PR makes **[no]** API changes.

### Testing Plan
- [x] I have tested Android, Java SE, and Java EE

### Test Steps
 - Comment out FOREGROUND_SERVICE permissions
 - Try to run the app connecting to TDK through Bluetooth
 - Ensure app is able to run without crashing

### Summary
This PR will ensure that permissions are checked before calling startForeground and catch exceptions thrown by startForeground

### CLA
